src/accountsdb/db.zig

```diff
@@ -3916,6 +4098,114 @@ test "shrink account file works" {
     account.deinit(allocator);
 }
 
+test "generate snapshot & update gossip snapshot hashes" {
+    // TODO: this overwrites the snapshots in the testdata folder as is, we need a mechanism
+    // to load from a snapshot outside of accountsdb's working snapshot directory.
+    if (true) return error.SkipZigTest;
```

Comment:
TODO: make this unit test not overwrite the archives in `data/test-data`, so it doesn't need to be skipped.